integrationtests/server-integration/server-integration-commons/src/test/java/org/infinispan/test/integration/remote/AbstractHotRodQueryIT.java

```diff
@@ -29,54 +31,73 @@
  */
 public abstract class AbstractHotRodQueryIT {
 
-   private RemoteCacheManager rcm;
+   private static final String HOST = "127.0.0.1";
 
-   private static RemoteCacheManager createCacheManager() {
-      return new RemoteCacheManager(createConfiguration(), true);
-   }
-
-   private static Configuration createConfiguration() {
-      ConfigurationBuilder config = new ConfigurationBuilder();
-      config.addServer().host("127.0.0.1");
-      config.marshaller(new ProtoStreamMarshaller());
-      return config.build();
-   }
-
-   @After
-   public void cleanUp() {
-      if (rcm != null)
-         rcm.stop();
+   @Test
+   public void testIndexed() {
+      BookQuerySchema schema = new BookQuerySchemaImpl();
+      ConfigurationBuilder config = localServerConfiguration();
+      config.addContextInitializer(schema);
+      try (RemoteCacheManager remoteCacheManager = new RemoteCacheManager(config.build())) {
+
+         // register schema
+         registerSchema(remoteCacheManager, schema.getProtoFileName(), schema.getProtoFile());
+
+         String xmlConfig = "" +
+               "<distributed-cache name=\"books\">\n" +
+               "  <indexing path=\"${java.io.tmpdir}/index\">\n" +
+               "    <indexed-entities>\n" +
+               "      <indexed-entity>" + Book.class.getName() + "</indexed-entity>\n" +
```

Comment:
```suggestion
               "      <indexed-entity>book_sample.Book</indexed-entity>\n" +
```

Reply:
I think I forgot to compile `mvn clean install -pl integrationtests/server-integration/server-integration-commons -s maven-settings.xml`